localstack/services/apigateway/provider.py

```diff
@@ -858,6 +863,15 @@ def get_stage(self, context: RequestContext, rest_api_id: String, stage_name: St
         self._patch_stage_response(response)
         return response
 
+    def get_stages(
+        self, context: RequestContext, rest_api_id: String, deployment_id: String = None
+    ) -> Stages:
+        response = call_moto(context)
+        for stage in response["item"]:
+            self._patch_stage_response(stage)
+            stage.pop("description", None)
```

Comment:
I wonder if the description should only be discarded if not present, it's in the model, just a case of remove if not present perhaps? https://docs.aws.amazon.com/apigateway/latest/api/API_GetStages.html#API_GetStages_ResponseElements

Reply:
Great catch, thanks for checking it! Updated the PR .. 👍